### PR TITLE
add homebrew stats

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function Comma(Num) { //function to add commas to textboxes
+  Num += '';
+  Num = Num.replace(',', '');
```
Comment (karianna) [inline common.js]: Could loop this?

File: assets/docker.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...25 49.543h19.818v19.818h-19.818V49.543zm1.65 1.652h1.565V67.71h-1.564V51.195zm2.94 0h1.627V67.71h-1.626V51.195zm3.004 0h1.627V67.71h-1.627V51.195zm3.002 0h1.627V67.71h-1.627V51.195zm3.004 0h1.626V67.71h-1.626V51.195zm3.002 0h1.564V67.71h-1.564V51.195z" fill="#23A3C2"/><path d="M96.704 101.492a5.468 5.468 0 1 1-.002 10.935 5.468 5.468 0 0 1 .002-10.935" fill="#D3ECEC"/><path d="M96.704 103.043c.5 0 .977.094 1.417.265a1.598 1.598 0 0 0 .798 2.98c.605 0 1.13-.335 1.402-.831a3.915 3.915 0 1 1-3.617-2.414M0 90.162h254.327c-5.537-1.404-17.52-3.302-15.544-10.56-10.07 11.652-34.353 8.175-40.482 2.43-6.824 9.898-46.554 6.135-49.325-1.576-8.556 10.041-35.067 10.041-43.623 0-2.773 7.711-42.502 11.474-49.327 1.575-6.128 5.746-30.41 9.223-40.48-2.428C17.522 86.86 5.539 88.758 0 90.163" fill="#364548"/><path d="M111.237 140.89c-13.54-6.425-20.971-15.16-25.106-24.694-5.03 1.435-11.075 2.353-18.1 2.747-2.646.148-5.43.224-8.35.224-3.368 0-6.917-.1-10.643-.297 12.417 12.41 27.692 21.964 55.976 22.138 2.088 0 4.16-.04 6.223-.118" fill="#BDD9D7"/><path d="M91.16 124.994c-1.873-2.543-3.69-5.739-5.026-8.8-5.03 1.437-11.077 2.355-18.103 2.75 4.826 2.619 11.727 5.046 23.13 6.05" fill="#D3ECEC"/></g></svg>
```
Comment (karianna) [inline assets/docker.svg:1]: Formatting?  Not that it really matters :-).

File: common.js (hunk 1 above)
Comment (karianna) [reply common.js]: Some better variable naming please, e.g. what is the Num for, what is x, x1, x2 really (if co-ordinates then what's different about x, x1, x2)

File: common.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  Num = Num.replace(',', '');
+  Num = Num.replace(',', '');
+  Num = Num.replace(',', '');
+  Num = Num.replace(',', '');
+  Num = Num.replace(',', '');
+  x = Num.split('.');
+  x1 = x[0];
+  x2 = x.length > 1 ? '.' + x[1] : '';
+  var rgx = /(\d+)(\d{3})/;
```
Comment (karianna) [inline common.js]: Example in comments of what you're pulling out

File: common.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+function Comma(Num) { //function to add commas to textboxes
```
Comment (karianna) [inline common.js]: rename function to addComma or some such.

File: homebrew.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        console.log('Looks like there was a problem. Status Code: ' +
+          response.status);
+        return;
+      }
+      response.text().then(function(data) {
+        // Grab all the cask files from the README.md
+        const regex = /\b(\w*adoptopenjdk\w*[-]?\w*)\b/gm
+        casks = data.match(regex);
+
+        fetch('https://formulae.brew.sh/api/analytics/cask-install/365d.json', {})
```
Comment (karianna) [inline homebrew.js:16]: Can some of this be split into actual functions?

**Review by karianna — CHANGES_REQUESTED** [review]

File: assets/docker.svg (hunk 2 above)
Comment (gdams) [reply assets/docker.svg:1]: don't want to modify someone else's logo really.

File: homebrew.js (hunk 5 above)
Comment (gdams) [reply homebrew.js:16]: I kind of wanted to keep this as a single logical function

File: common.js (hunk 4 above)
Comment (gdams) [reply common.js]: done

**Comment by gdams** [comment]
@karianna updated PTAL

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**